src/interfaces/banner.ts

```diff
@@ -0,0 +1,4 @@
+export type Banner = string | boolean | {
```

Comment:
Can be further simplified to just { text, href }. Can either make href optional or always set href in the post. Don’t need Boolean since you Banner is already set as an optional type, so it can be undefined

Reply:
Alright, I'll work on it after work. `banner: false` actually overrides any previous values and hides the banner, whereas leaving it undefined causes it to be ignored. I kinda liked this behavior but I can replace it with a `hideBanner` flag for instance.